hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd

```diff
@@ -2020,4 +2024,46 @@
             <xs:enumeration value="POOLED"/>
         </xs:restriction>
     </xs:simpleType>
+
+
+    <xs:complexType name="quorum">
+        <xs:sequence>
```

Comment:
this should be `xs:all` rather than `xs:sequence` to provide more relax configuration to user.